README.md

```diff
@@ -7,4 +7,44 @@ conda activate dope
 
 pip install -r requirements.txt 
 pip install -e .  
+
+```
+### Worker Deployment using Docker
+```
+Standard Worker Example
+To adapt to other workers, follow same convention, just cd to the respective folder
+First, 
+
+    cd /docker/standard_worker/
```

Comment:
```suggestion
    cd ./docker/standard_worker/
```

Reply:
> Let's specify things in a bit more details:
> 
> 1. How does the pipeline operate?
> 2. What happen, what does it mean to launch the worker(s) with `docker compose up -d`?
> 3. Specify that all of the MATLAB and `ecephys_directory` parts in the `.env` are only relevant for ephys spike sorting worker, no other workers need this. (perhaps it makes sense to have multiple example.env files.
> 4. Are you mixing up the `.env` and `build.env` files? They are for different purpose, right? The `build.env` should have very minimal content in it

Ok sounds good, I wasn't sure how much detail to go into here, but I can describe the process of all the things happening with each of the commands. Yes I can modify the example.env's so that they are specific to the worker... will remove all MATLAB stuff from the non spike-soritng workers. I took the assumption that the build.env is the template for the .env file, but if we use the build.env elsewhere, I can add an example.env separately